src/library_gl.js

```diff
@@ -767,16 +777,6 @@ var LibraryGL = {
 
       var GLctx = context.GLctx;
 
-      context.maxVertexAttribs = GLctx.getParameter(GLctx.MAX_VERTEX_ATTRIBS);
```

Comment:
Here I could not see a reason why the above line should be outside the `FULL_ES2=1` block, so put it inside when moving it.